src/applications/disability-benefits/all-claims/config/8940/index.js

```diff
@@ -184,6 +186,7 @@ export default function() {
           properties: {},
```

Comment:
conclusion4192 is still in `8940/index.js`, should be moved to `4142/index.js`

Reply:
moved!